src/music/player.py

```diff
@@ -297,10 +298,11 @@ async def play_youtube(self, link: str):
                         raw_audio_source = discord.FFmpegPCMAudio(executable=FFMPEG_PATH, source=source, **self.FFMPEG_OPTIONS)
                         audio = MusicSource(raw_audio_source, info = video_info, volume= self.volume/100)
                         self.playlist.add(audio)
-                        self.paused = False
 
+                        # If the player is not playing because it just came in to the channel (not because of being paused), advance the track head to the next (just added) song
                         if not self.connected_client.is_playing():
```

Comment:
can this not be done using an and statement?

Reply:
Do you mean like this? It's not a problem, whichever is the most clear.
```suggestion
                        if not self.connected_client.is_playing() and not self.connected_client.is_paused():
```

Reply:
yup I prefer it be more concise, the lest nested something is the better in my opinion